src/appleseed/renderer/kernel/shading/oslshadergroupexec.cpp

```diff
@@ -51,7 +51,10 @@ OSLShaderGroupExec::OSLShaderGroupExec(OSL::ShadingSystem& shading_system)
   : m_osl_shading_system(shading_system)
   , m_osl_thread_info(shading_system.create_thread_info())
   , m_osl_shading_context(shading_system.get_context(m_osl_thread_info))
+  , m_osl_mem_used(0)
 {
+    m_osl_mem_pool = new char[OSLMemPoolSize + OSLMemAlignment];
```

Comment:
If the size is known at compile-time, can't this be statically allocated in the class?